templates/registration/login.html

```diff
@@ -0,0 +1,21 @@
+{% extends "base.html" %}
+{% block content %}
+  {% load crispy_forms_tags %}
+  {% if form.errors %}
+  <p>Your username and password didn't match. Please try again.</p>
+  {% endif %}
+
+  <h1>Login</h1>
+
+  {% if next %}
+    <p>Please login to see this page.</p>
+  {% endif %}
```

Comment:
The 'next' parameter should be checked for safety to prevent open redirect vulnerabilities. Consider using the 'is_safe_url' utility from Django to ensure the 'next' URL is safe.